src/main/java/org/usfirst/frc/team4099/robot/subsystems/Drive.kt

```diff
@@ -34,16 +44,129 @@ class Drive private constructor() : Subsystem {
         leftSlave1SRX.set(ControlMode.Follower, Constants.Drive.LEFT_MASTER_ID.toDouble())
         leftSlave2SRX.set(ControlMode.Follower, Constants.Drive.LEFT_MASTER_ID.toDouble())
         leftMasterSRX.set(ControlMode.PercentOutput, 0.0)
+        leftMasterSRX.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder, 0, 0)
+        leftMasterSRX.setSensorPhase(false)
+        leftMasterSRX.setStatusFramePeriod(StatusFrameEnhanced.Status_2_Feedback0, 5, 0)
+        leftMasterSRX.config_kP(0, Constants.Gains.LEFT_LOW_KP, 0)
+        leftMasterSRX.config_kI(0, Constants.Gains.LEFT_LOW_KI, 0)
+        leftMasterSRX.config_kD(0, Constants.Gains.LEFT_LOW_KD, 0)
+        leftMasterSRX.config_kF(0, Constants.Gains.LEFT_LOW_KF, 0)
+
+        leftMasterSRX.config_kP(1, Constants.Gains.LEFT_HIGH_KP, 0)
+        leftMasterSRX.config_kI(1, Constants.Gains.LEFT_HIGH_KI, 0)
+        leftMasterSRX.config_kD(1, Constants.Gains.LEFT_HIGH_KD, 0)
+        leftMasterSRX.config_kF(1, Constants.Gains.LEFT_HIGH_KF, 0)
+
+
 
         rightSlave1SRX.set(ControlMode.Follower, Constants.Drive.RIGHT_MASTER_ID.toDouble())
         rightSlave2SRX.set(ControlMode.Follower, Constants.Drive.RIGHT_MASTER_ID.toDouble())
         rightMasterSRX.set(ControlMode.PercentOutput, 0.0)
+        rightMasterSRX.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder, 0, 0)
```

Comment:
Can you put (in a comment) what some of these lines do? it's not immediately obvious what, for example, `setSensorPhase` does